lib/auth.ts

```diff
@@ -457,18 +457,12 @@ const getAuth = function (
 	 * balena.auth.register({
 	 * 	email: 'johndoe@gmail.com',
 	 * 	password: 'secret'
-	 * }).then(function(token) {
-	 * 	console.log(token);
+	 * }).then(function(result) {
+	 * 	console.log(`Token: ${result}`);
+	 * }).catch(function(error) {
+	 * 	throw error;
```

Comment:
I don't see a reason for these changes. What is it you mean to convey to the user with the addition of the catch handler that merely rethrows the error?

Reply:
The reason I added them first was to cover both branches. But makes sense to not show the err' if we just show an example of throwing it again. Updated the PR.